src/components/atoms/MenuDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactElement } from 'react'
+import Tooltip from './Tooltip'
+import styles from './MenuDropdown.module.css'
+import { ReactComponent as Caret } from '../../images/caret.svg'
+import LinkOpener from '../molecules/LinkOpener'
+import classNames from 'classnames/bind'
+
+const cx = classNames.bind(styles)
+
+declare type MenuItem = {
+  name: string
+  link?: string
+}
+
+interface MenuNestedItem extends MenuItem {
+  subItems?: MenuItem[]
+}
+
+function ItemLabel({
+  name,
+  className
+}: {
+  name: string
+  className?: string
+}): ReactElement {
+  return (
+    <div className={cx({ menuItem: true, [className]: className })}>
+      {name}
+      <Caret aria-hidden="true" className={styles.caret} />
+    </div>
+  )
+}
+
+export default function MenuDropdown({
+  label,
+  items
+}: {
+  label: string
+  items: MenuNestedItem[]
+}): ReactElement {
+  return (
+    <Tooltip
+      content={
+        <ul>
+          {items.map((item, i) => (
+            <li key={`${item.name}-${i}`}>
+              {item?.subItems && item.subItems.length > 0 ? (
+                <Tooltip
+                  content={
+                    <ul>
+                      {item.subItems.map((subItem, i) => (
+                        <li key={`${subItem.name}-${i}`}>
+                          <LinkOpener
+                            uri={subItem.link}
+                            className={styles.link}
+                          >
+                            {subItem.name}
+                          </LinkOpener>
+                        </li>
+                      ))}
```

Comment:
suggestion(blocking): I feel like this could probably be refactored into its own component. Something like:
```jsx
<ul>
  <li key={`${item.name}-${i}`}>
    {item.subItems ?
      <Tooltip ... content={<ItemComponent subitems={item.subitems}} /> :
      <LinkOpener
        uri={item.link}
        className={styles.link}
      >
        {item.name}
      </LinkOpener>
    }
  </li>
</ul>

```

Did you consider this, or could have a look what is possible here to reduce the code duplication?